pkgs/development/python-modules/dictances/default.nix

```diff
@@ -0,0 +1,27 @@
+{ buildPythonPackage
+, fetchPypi
+, isPy3k
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "dictances";
+  version = "1.5.1";
+
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1xc89j6n5j9i8jk6qv2y87wcny3aq5g06w26b91z1ycm8mkyayzy";
+  };
+
+  # Missing some nixpkgs
+  doCheck = false;
```

Comment:
I would still like to see some tests, they are really important to demonstrate that a package isn't broken when it's dependencies get bumped.... although I guess this package doesn't really have any direct dependencies :)

Still would be nice :D